includes/Admin/Admin_Page.php

```diff
@@ -49,13 +57,27 @@ public function add_hooks() {
 	 * @since n.e.x.t
 	 */
 	public function add_page() {
-		$hook = add_management_page(
-			__( 'Plugin Check', 'plugin-check' ),
-			__( 'Plugin Check', 'plugin-check' ),
-			'activate_plugins',
-			'plugin-check',
-			array( $this, 'render_page' )
-		);
+
+		if ( is_multisite() ) {
+
+			$hook = add_submenu_page(
+				'settings.php',
+				__( 'Plugin Check', 'plugin-check' ),
+				__( 'Plugin Check', 'plugin-check' ),
+				'activate_plugins',
```

Comment:
```suggestion
				'manage_network_plugins',
```

[manage_network_plugins](https://wordpress.org/documentation/article/roles-and-capabilities/#manage_network_plugins)